table-attributes.md

```diff
@@ -73,47 +75,49 @@ alter table t partition p attributes[=]'key=value1';
 
 ### 使用方法
 
+使用方法如下，其中 `t` 为所要修改的表名，`p` 为所要修改的分区名。
+
 + 禁止属于某个表的 Region 被合并
 
     ```sql
-    alter table t attributes[=]'merge_option=deny';
+    ALTER TABLE t ATTRIBUTES 'merge_option=deny';
     ```
 
 + 允许属于某个表的 Region 被合并
 
     ```sql
-    alter table t attributes[=]'merge_option=allow';
+    ALTER TABLE t ATTRIBUTES 'merge_option=allow';
     ```
 
 + 重置某个表的属性
 
     ```sql
-    alter table t attributes[=]default；
+    ALTER TABLE t ATTRIBUTES default;
```

Comment:
Does `default` need to use uppercase? @dveeden

Reply:
according to changes in https://github.com/pingcap/docs/pull/6950, the "default" here needs to use uppercase.

Reply:
```suggestion
    ALTER TABLE t ATTRIBUTES DEFAULT;
```